include/bout/paralleltransform.hxx

```diff
@@ -92,6 +94,11 @@ public:
   bool canToFromFieldAligned() override{
     return true;
   }
+
+  DIRECTION getDefaultYDirectionType() const override {
+    return DIRECTION::YAligned;
```

Comment:
I'm not sure about this at all. This says that the identity transform is synonymous with a field-aligned grid, and I'm not convinced that's true.

What does saying the identity transform has field-aligned fields by default get us?